### PR TITLE
Testing single versions for container and openshift tests

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Comment by phracek** [comment]
[test-all]

File: .github/workflows/container-tests.yml [hunk 1]
```diff
@@ -76,15 +84,39 @@ jobs:
...
+        shell: bash
+        id: vars
+        run: |
+          dockerfile="Dockerfile.${{ matrix.os_test }}"
+          if [[ ${{ matrix.os_test }} == "centos7" ]]; then
+            dockerfile="Dockerfile"
+          fi
+          echo "::set-output name=DOCKERFILE_NAME::${dockerfile}"
+
+      - name: Check ${{ matrix.version }}/${{ steps.vars.outputs.DOCKERFILE_NAME }} presents
```
Comment (zmiklank) [inline .github/workflows/container-tests.yml:97]:
```suggestion
      - name: Check that ${{ matrix.version }}/${{ steps.vars.outputs.DOCKERFILE_NAME }} is present
```

**Review by zmiklank — APPROVED** [review]
LGTM. But please, in general, do not push commits with names such as "[Update .github/workflows/container-tests.yml](https://github.com/sclorg/redis-container/pull/129/commits/32fe8222be8243d3c2e5c6448eea8ad8b0f8f577)" to master as they are non-explanatory.

**Comment by phracek** [comment]
[test-all]

**Pull request merged**